### ISSUE TITLE
expire returned can be wrong with never expires

### ISSUE THREAD

**Issue by absolute8511** [aerospike/aerospike-client-go#257]
The expire in Record returned sometimes is nerver expire while the record is nearly expire. I saw the Java has the below code to fix this. Maybe this should be fixed in Go also.

```
		// Record may not have expired on server, but delay or clock differences may
		// cause it to look expired on client. Floor at 1, not 0, to avoid old
		// "never expires" interpretation.
		return (expiration < 0 || expiration > now) ? expiration - now : 1;
```

**Comment by khaf** [comment]
Makes sense. Will be fixed in the next release. Thanks.

**Comment by khaf** [comment]
This was fixed in v2.1.0. Thanks for your report.
